internal/infrastructure/mongo/asset_test.go

```diff
@@ -0,0 +1,108 @@
+package mongo
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/reearth/reearth-backend/internal/infrastructure/mongo/mongodoc"
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.mongodb.org/mongo-driver/x/mongo/driver/uuid"
+)
+
+func TestFindByID(t *testing.T) {
+	// Skip unit testing if "REEARTH_DB" is not configured
+	// See details: https://github.com/reearth/reearth/issues/273
+	db := os.Getenv("REEARTH_DB")
+	if db == "" {
+		return
+	}
+
+	type a struct {
+		id          asset.ID
+		createdAt   time.Time
+		team        asset.TeamID
+		name        string
+		size        int64
+		url         string
+		contentType string
+	}
+
+	tests := []struct {
+		Name     string
+		Expected struct {
+			Name  string
+			Asset *a
+		}
+	}{
+		{
+			Expected: struct {
+				Name  string
+				Asset *a
+			}{
+				Asset: &a{
+					id:          id.NewAssetID(),
+					createdAt:   time.Now(),
+					team:        id.NewTeamID(),
+					name:        "name",
+					size:        10,
+					url:         "hxxps://xxx.com",
+					contentType: "json",
+				},
+			},
+		},
+	}
+
+	c, _ := mongo.Connect(
+		context.Background(),
+		options.Client().
+			ApplyURI(db).
+			SetConnectTimeout(time.Second*10),
+	)
+
+	for _, tc := range tests {
+		tc := tc
+
+		t.Run(tc.Name, func(t *testing.T) {
+			t.Parallel()
+
+			want, err := asset.New().
+				NewID().
+				Team(tc.Expected.Asset.team).
+				Name(tc.Expected.Asset.name).
+				Size(tc.Expected.Asset.size).
+				URL(tc.Expected.Asset.url).
+				CreatedAt(tc.Expected.Asset.createdAt).
+				Build()
+
+			assert.NoError(t, err)
+
+			database, _ := uuid.New()
```

Comment:
Note 📖 

I used the Mongo's (`go.mongodb.org/mongo-driver/x/mongo/driver/uuid`) `uuid` generator as the database name instead of the Google ones (`github.com/google/uuid`) which are more popular. This is because since we use Mongo already, we won't need to add additional packages.